packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -256,6 +259,7 @@ export class AiAgentModel {
                     name: args.name,
                     project_uuid: args.projectUuid,
                     tags: args.tags,
+                    updated_at: trx.fn.now() as unknown as Date,
```

Comment:
you can follow the same behaviour in other parts of the codebase with

```suggestion
                    updated_at: new Date()
```